src/server.cpp

```diff
@@ -917,9 +919,19 @@ void Server::AsyncRunStep(bool initial_step)
 						&far_players, disable_single_change_sending ? 5 : 30);
 				break;
 			case MEET_BLOCK_NODE_METADATA_CHANGED:
-				infostream << "Server: MEET_BLOCK_NODE_METADATA_CHANGED" << std::endl;
+				{
```

Comment:
You don't need the brace and two indentation levels. Keep it indented one level only, its better for readability.

Also, add a profiler line like `MEET_REMOVENODE` or `MEET_SWAPNODE` have it.